URLHaus - adding retry mechanism (#6965)
demisto/content: Packs/URLHaus/Integrations/URLHaus/CHANGELOG.md

## [Unreleased]

> - Added *Number of retries* instance parameter to determine how many times should a command be attempted before raising an error.

Reviewer: ```suggestion
- Added the *Number of retries* parameter which determines how many times a command should attempt before raising an error.
```

> - Fixed an issue where ***urlhaus-download-sample*** command would raise an error in case on results were found.

Reviewer: ```suggestion
- Fixed an issue where the ***urlhaus-download-sample*** command would raise an error in cases where results were found.
```
